.github/workflows/pr_comment_commands.yml

```
@@ -0,0 +1,89 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: PR commands
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: read
+  pull-requests: write
+  actions: write
+  checks: write
+
+jobs:
+
+  run_extended_tests:
```

Can we please add a human summary here of what this does -- something like
```suggestion
  # Starts the extended_tests when someone leaves a `Run extended tests` comment
  run_extended_tests:
```

It might also be nice to make the comparison case insensitive (so `run extended tests` would also work)

---

Added the comment 👍

---

As for the case sensitivity, maybe let's add it in a follow-up PR since changing the triggers will require a whole new round of testing to make sure the condition work as expected again